cfme/configure/access_control.py

```diff
@@ -1289,7 +1289,9 @@ class DetailsTenantView(ConfigurationView):
     name = Text('Name')
     description = Text('Description')
     parent = Text('Parent')
-    table = Table('//*[@id="fieldset"]/table')
+    table = Table(VersionPick(
+        {Version.lowest(): '//*[@id="fieldset"]/table',
```

Comment:
REQUIRED: I believe you can get rid of VersionPick here by using something like
```
//*[self::fieldset or @id="fieldset"]/table
```

Reply:
Depends how many things match that xpath, in the case of fieldset tag I would not be so sure about uniqueness, I prefer an ID in the path :)

Reply:
@mfalesni,  as far as I see old div with fieldset id turned into fieldset class. So, I doubt there are several fieldset classes on the page.
One more note, such long xpath with indexes and etc doesn't look robust.

Reply:
I checked on all z-streams, this table is the only one having `fieldset` on the page